CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 ### Changes
-- refactorings #3373 #3345
+- refactorings #3373 #3345 #3382
```

Comment:
might as well do `#skip-changelog` and omit the merge conflicts, but whatever you prefer.